frontend/tests/Factories/Problem.php

```diff
@@ -140,6 +147,7 @@ public static function getRequest(?\OmegaUp\Test\Factories\ProblemParams $params
             'show_diff' => $params->showDiff,
             'allow_user_add_tags' => $params->allowUserAddTags,
             'problem_level' => 'problemLevelBasicIntroductionToProgramming',
+            'public_tags' => 'problemTagBinarySearchTree',
```

Comment:
Acá como `selectedTags` ya forma parte de tu clase `ProblemParams`, puedes mandarlo llamar así:
```suggestion
            'public_tags' => $params->selectedTags,
```